src/main/java/me/totalfreedom/totalfreedommod/command/Command_unbanip.java

```diff
@@ -54,9 +54,12 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
 
         if (!silent)
         {
-            FUtil.staffAction(sender.getName(), "Unbanned the ip " + ip, true);
+            FUtil.staffAction(sender.getName(), "Removed the ip " + ip + "from the ban list.", true);
+        }
+        else
+        {
+            msg("Quietly removed the ip " + ip + " from the ban list.");
```

Comment:
Probably worth using the term silently given that's the -s flag

Reply:
what? it says -q, not -s